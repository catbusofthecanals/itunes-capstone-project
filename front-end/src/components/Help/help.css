.help-container .back-btn,
.help-container .blank {
  width: 120px;
}

.help-container {
  text-align: center;
}

.help {
  padding-top: 5%;
  padding-bottom: 10%;
  margin-bottom: 10%;
}

img.logo {
  width: 25px;
}

h4 {
  margin-top: 5%;
}

p {
  font-size: 1em;
  line-height: 1em;
}
