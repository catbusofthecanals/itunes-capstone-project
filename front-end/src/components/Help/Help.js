import React from "react";
import { Link } from "react-router-dom";
import Bin from "./trash-can.png";
import Star from "./greystar.png";
import FavStar from "./star.png";
import "./help.css";

const Help = () => {
  return (
    <div className="help-container">
      <div className="header-section">
        <div className="header-container">
          <div className="back-btn">
            {/* Link back to Home */}
            <Link to="/">Back</Link>
          </div>
          <div className="header">Help</div>
          {/* use to keep heading centred */}
          <div className="blank"></div>
        </div>
      </div>
      <div className="help">
        <div className="help-search">
          <h4>Search for an item:</h4>
          <p>Enter a search term in the search box</p>
          <p>
            Filter your search by media type from the drop-down menu or leave as
            'All Media' to search all formats
          </p>
          <p>Press 'Search'</p>
        </div>
        <div className="help-add-fav">
          <h4>Add to favourites:</h4>
          <p>Find the item you would like to add to your favourites</p>
          <p>
            Select the <img className="logo" src={Star} alt="Favourite" /> icon
            next to that item to add it to your favourites
          </p>
        </div>
        <div className="help-view-fav">
          <h4>View favourites:</h4>
          <p>
            In the header bar, select the 'Favourites' button to see a list of
            your favourited items
          </p>
        </div>
        <div className="help-remove-fav">
          <h4>Remove an item within favourites:</h4>
          <p>
            {" "}
            Select the <img className="logo" src={Bin} alt="Trash Can" /> icon
            next to the item you would like to remove from your favourites
          </p>
        </div>
        <div className="help-remove-fav-home">
          <h4>Remove an item from favourites from the search page:</h4>
          <p>
            If an item has a{" "}
            <img className="logo" src={FavStar} alt="Remove Favourite" /> icon
            then it is in your favourites.
          </p>
          <p>Select the icon to remove this item from your favourites</p>
        </div>
      </div>
    </div>
  );
};

export default Help;
