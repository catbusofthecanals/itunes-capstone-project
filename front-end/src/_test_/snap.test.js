import React from "react";
import Help from "../components/Help/Help";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";
//import BrowserRouter to successfully render <Link> in heading

// snapshot test of help component
test("renders correctly", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Help />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
