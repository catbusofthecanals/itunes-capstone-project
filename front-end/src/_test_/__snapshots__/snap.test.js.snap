// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="help-container"
>
  <div
    className="header-section"
  >
    <div
      className="header-container"
    >
      <div
        className="back-btn"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          Back
        </a>
      </div>
      <div
        className="header"
      >
        Help
      </div>
      <div
        className="blank"
      />
    </div>
  </div>
  <div
    className="help"
  >
    <div
      className="help-search"
    >
      <h4>
        Search for an item:
      </h4>
      <p>
        Enter a search term in the search box
      </p>
      <p>
        Filter your search by media type from the drop-down menu or leave as 'All Media' to search all formats
      </p>
      <p>
        Press 'Search'
      </p>
    </div>
    <div
      className="help-add-fav"
    >
      <h4>
        Add to favourites:
      </h4>
      <p>
        Find the item you would like to add to your favourites
      </p>
      <p>
        Select the 
        <img
          alt="Favourite"
          className="logo"
          src="greystar.png"
        />
         icon next to that item to add it to your favourites
      </p>
    </div>
    <div
      className="help-view-fav"
    >
      <h4>
        View favourites:
      </h4>
      <p>
        In the header bar, select the 'Favourites' button to see a list of your favourited items
      </p>
    </div>
    <div
      className="help-remove-fav"
    >
      <h4>
        Remove an item within favourites:
      </h4>
      <p>
         
        Select the 
        <img
          alt="Trash Can"
          className="logo"
          src="trash-can.png"
        />
         icon next to the item you would like to remove from your favourites
      </p>
    </div>
    <div
      className="help-remove-fav-home"
    >
      <h4>
        Remove an item from favourites from the search page:
      </h4>
      <p>
        If an item has a
         
        <img
          alt="Remove Favourite"
          className="logo"
          src="star.png"
        />
         icon then it is in your favourites.
      </p>
      <p>
        Select the icon to remove this item from your favourites
      </p>
    </div>
  </div>
</div>
`;
